### DISCUSSION TITLE
need to show product description and varients when a particular product is selected

### DISCUSSION THREAD

**Discussion by vishnupriyakem** [vercel/next.js#60561] [Help] [answered]
### Summary

when i click on a particular products i can see only the title, prize and add to cart but not able to see the product description ,when i console log description, descriptionHtml is null what to do?

producy {
  id: 'gid://shopify/Product/7898192085142',
  handle: 'the-collection-snowboard-liquid',
  availableForSale: true,
  title: 'The Collection Snowboard: Liquid',
  description: '',
  descriptionHtml: '',
  options: [
  {
  id: 'gid://shopify/ProductOption/10028217893014',
  name: 'Title',
  values: [ 'Default Title' ]
}
],
  priceRange: {
  maxVariantPrice: { amount: '749.95', currencyCode: 'INR' },
  minVariantPrice: { amount: '749.95', currencyCode: 'INR' }
},
  featuredImage: {
  url: 'https://cdn.shopify.com/s/files/1/0628/7530/4086/products/Main_b13ad453-477c-4ed1-9b43-81f3345adfd6.jpg?v=1704172901',
  altText: 'Top and bottom view of a snowboard. The top view shows a stylized scene of water, trees, mountains,\n        sky and a moon in blue colours. The bottom view has a blue liquid, drippy background with the text “liquid” in\n        a stylized script typeface.',
  width: 1600,
  height: 1600
},
  seo: { description: null, title: null },
  tags: [ 'Accessory', 'Sport', 'Winter' ],
  updatedAt: '2024-01-10T05:43:14Z',
  images: [
  {
  url: 'https://cdn.shopify.com/s/files/1/0628/7530/4086/products/Main_b13ad453-477c-4ed1-9b43-81f3345adfd6.jpg?v=1704172901',
  altText: 'Top and bottom view of a snowboard. The top view shows a stylized scene of water, trees, mountains,\n        sky and a moon in blue colours. The bottom view has a blue liquid, drippy background with the text “liquid” in\n        a stylized script typeface.',
  width: 1600,
  height: 1600
}
],
  variants: [
  {
  id: 'gid://shopify/ProductVariant/42999768383638',
  title: 'Default Title',
  availableForSale: true,
  selectedOptions: [ { name: 'Title', value: 'Default Title' } ],
  price: { amount: '749.95', currencyCode: 'INR' }
}
]
}

**Comment by icyJoseph** [comment, marked as answer]
I guess, this is an issue with your CSM/DB and how you query the description from it? Could you share more information about what is it you are doing?

**Reply by vishnupriyakem** [reply]
thank you for your response
before it was /api/2023-01/  when i changed to =>/api/2023-10 it worked  
snippet
SHOPIFY_GRAPHQL_API_ENDPOINT = '/api/2023-10/graphql.json';
